packages/manager/modules/nutanix/src/dashboard/component/order-private-bandwidth/order-private-bandwidth.controller.js

```diff
@@ -0,0 +1,179 @@
+    this.steps = [
+      {
+        isValid: () => this.model.plan,
+        isLoading: () => this.isLoading,
+        load: () => {
+          this.isLoading = true;
+          return this.NutanixClusterOrderPrivateBandwidthService.getNutanixClusterPrivateBandwidthOptions(
+            this.serviceId,
+          )
+            .then((plans) => {
+              this.plans = NutanixClusterOrderPrivateBandwidthCtrl.getValidBandwidthPlans(
+                plans,
+              );
+              this.plans.sort(
+                (a, b) =>
+                  a.prices.find((el) => el.capacities.includes('renew'))
+                    .priceInUcents -
+                  b.prices.find((el) => el.capacities.includes('renew'))
+                    .priceInUcents,
+              );
+            })
+            .catch((error) => {
+              this.handleError(
+                error,
+                this.$translate.instant(
+                  'cluster_error_bandwidth_order',
+                  error.data,
+                ),
+              );
+              this.goBack();
+            })
+            .finally(() => {
+              this.isLoading = false;
+            });
+        },
+      },
+      {
+        isValid: () => this.model.plan,
+        isLoading: () => this.isLoading,
+        load: () => {
+          this.isLoading = true;
+          return this.NutanixClusterOrderPrivateBandwidthService.getNutanixClusterPrivateBandwidthOrder(
+            this.serviceId,
+            this.getPlanDetailsForPayload(),
+            this.numberOfNodes,
+          )
+            .then((res) => {
+              res.bandwidth = find(this.plans, {
+                planCode: this.model.plan,
+              }).bandwidth;
+              res.planCode = this.model.plan;
+              this.provisionalPlan = res;
+            })
+            .catch((error) => {
+              this.handleError(
+                error,
+                this.$translate.instant(
+                  'cluster_error_bandwidth_order',
+                  error.data,
+                ),
+              );
+              this.goBack();
+            })
+            .finally(() => {
+              this.isLoading = false;
+            });
+        },
+      },
+    ];
+  }
+
+  initFirstStep() {
+    this.steps[0].load();
+  }
+
+  initSecondStep() {
+    this.steps[1].load();
+  }
+
+  cancel() {
+    this.atInternet.trackClick({
+      name: `${this.trackingPrefix}cancel`,
+      type: 'action',
+    });
+    this.goBack();
+  }
+
+  order() {
+    if (this.model.plan) {
+      this.isLoading = true;
+      this.atInternet.trackClick({
+        name: `${this.trackingPrefix}confirm`,
+        type: 'action',
+      });
+      return this.NutanixClusterOrderPrivateBandwidthService.nutanixClusterPrivateBandwidthPlaceOrder(
+        this.serviceId,
+        this.getPlanDetailsForPayload(),
+        this.numberOfNodes,
+        this.region === 'US' || this.model.autoPayWithPreferredPaymentMethod,
```

Comment:
nitpick :
FYI we have coreConfig.isRegion (see : https://github.com/ovh/manager/blob/8b2ed07ecbb51c71545bff69f232597d75a9c533/packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js#L79)